src/plugins/files/server/blob_storage_service/adapters/es/content_stream/content_stream.test.ts

```diff
@@ -43,124 +46,193 @@ describe('ContentStream', () => {
   });
 
   describe('read', () => {
-    beforeEach(() => {
-      stream = getContentStream({ params: { size: 1 } });
-    });
+    describe('with `indexIsAlias` set to `true`', () => {
+      let searchResponse: estypes.SearchResponse<FileDocument<{}>>;
+
+      beforeEach(() => {
+        searchResponse = {
+          took: 3,
+          timed_out: false,
+          _shards: {
+            total: 2,
+            successful: 2,
+            skipped: 0,
+            failed: 0,
+          },
+          hits: {
+            total: {
+              value: 1,
+              relation: 'eq',
+            },
+            max_score: 0,
+            hits: [
+              {
+                _index: 'foo',
+                _id: '123',
+                _score: 1.0,
+              },
+            ],
+          },
+        };
+
+        client.search.mockResolvedValue(searchResponse);
+      });
 
-    it('should perform a search using index and the document id', async () => {
-      await new Promise((resolve) => stream.once('data', resolve));
+      it('should use es.search() to find chunk index', async () => {
+        stream = getContentStream({ params: { size: 1 }, indexIsAlias: true });
+        const data = await new Promise((resolve) => stream.once('data', resolve));
+
+        expect(client.search).toHaveBeenCalledWith({
+          body: {
+            _source: false,
+            query: {
+              term: {
+                _id: 'something.0',
+              },
+            },
+            size: 1,
+          },
+          index: 'somewhere',
+        });
+        expect(data).toEqual(Buffer.from('some content'));
+      });
 
-      expect(client.get).toHaveBeenCalledTimes(1);
+      it('should throw if chunk is not found', async () => {
+        searchResponse.hits.hits = [];
+        stream = getContentStream({ params: { size: 1 }, indexIsAlias: true });
 
-      const [[request]] = client.get.mock.calls;
-      expect(request).toHaveProperty('index', 'somewhere');
-      expect(request).toHaveProperty('id', 'something.0');
-    });
+        const readPromise = new Promise((resolve, reject) => {
+          stream.once('data', resolve);
+          stream.once('error', reject);
+        });
 
-    it('should read the document contents', async () => {
-      const data = await new Promise((resolve) => stream.once('data', resolve));
-      expect(data).toEqual(Buffer.from('some content'));
+        await expect(readPromise).rejects.toHaveProperty(
+          'message',
+          'Unable to determine index for file chunk id [something.0] in index (alias) [somewhere]'
+        );
+      });
     });
 
-    it('should be an empty stream on empty response', async () => {
-      client.get.mockResponseOnce(toReadable());
-      const onData = jest.fn();
+    describe('with `indexIsAlias` set to `false`', () => {
```

Comment:
From here down, everything is the same as before. All I did was wrap the test cases in a `describe()` whose title is more descriptive of that block of tests.

Here is a screen capture from my IDE: 

<img width="2294" alt="image" src="https://user-images.githubusercontent.com/56442535/228256595-a9f0db93-5797-417a-9883-ee439be35e89.png">